src/datagrid/DataGridFilter.tsx

```diff
@@ -156,11 +169,13 @@ export class DataGridFilter extends React.PureComponent<DataGridFilterProps, Dat
                     className={classNames([ClassNames.DATARID_FILTER, ClassNames.CLR_POPOVER_CONTENT, className])}
                     style={{
```

Comment:
Do we need to move this style to classes?

Reply:
@alexander-rassanov  I can not convert this into CSS class as we are not creating new CSS classes for our common components but using VMware Clarity CSS. Also I need to calculate value for 'transform' based on parent element.So keeping this in style.